FF8.Core/Sources/AreaName.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+// ReSharper disable StringLiteralTypo
+
+namespace FF8.Core
+{
+    public static class AreaName
+    {
+        public static String Get(Int32 areaId)
+        {
+            if (areaId >= 0 && areaId < _names.Length)
+                return _names[areaId];
+
+            return $"Unknown area: {areaId}";
+        }
+
+        private static readonly String[] _names =
+        {
+            /* [000] */ "???",
```

Comment:
I think these names (which are tied to world areas) are in wmsetXX.obj file, therefore it should be eventually replaced with area names parsed from the file instead

Reply:
this should be similar to the locationid names as saves from worldmap display correct names just like the ones from fields.